HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.10.8
```

Comment:
I think we typically wait for a full release before we put something in the release history - work with @digimaun on when/how to add release notes

Reply:
This is OK - we now like to update the release history (if there are user facing, or otherwise notable changes) per PR. It helps to more quickly inform the GitHub release notes.